### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by amirfz**

### CODE REVIEW

**Comment by amirfz** [comment]
@oshoma re issue #32

**Review by oshoma — COMMENTED** [review]
1. I suggest adding a line above the MIT license clarifying which directories it applies to. Similar to your comment about `/doc` for CC license. 

2. Unlike the MIT License, CC 1.0 Universal does not require that that the CC0 license and authorship information be retained in forks of this work. So in theory, someone could fork the docs in this repo and do whatever they want with them, with no mention of any of the original contributors, and strip off the original license. If you want the permissions for the docs to be similar to those for the code, I suggest instead you use CC-BY-SA, or perhaps CC-BY. See here: https://creativecommons.org/licenses/ under "The Licenses".

**Comment by amirfz** [comment]
@oshoma updated based on your feedback

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+=======
+
+Content under `/doc` is provided under Creative Common License as follows:
+
+**Creative Commons Legal Code**
+
+# Creative Commons Attribution-ShareAlike 4.0 International
```
Comment (oshoma) [inline LICENSE.md:31]: I think you can delete lines 31-41 inclusive. As I understand it this is explanatory preamble from Creative Commons, not the actual text of the license. 

You could also add a reference to the source e.g. "For more information about this license see https://creativecommons.org/licenses/by-sa/4.0/legalcode".

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
+Content under `/apps` is provided under MIT License as follows:
```
Comment (oshoma) [inline LICENSE.md:1]: suggestion:

> The content of this repository, other than the `/doc` directory which is licensed separately below, is provided under MIT License as follows:

**Comment by amirfz** [comment]
@oshoma could you approve the PR so that I can merge it?

**Comment by oshoma** [comment]
I approved already but I think it has no effect because I'm not a member of Aggregate Intellect organization.

**Comment by amirfz** [comment]
I see. @20001LastOrder @saminegash could you please approve and merge this?

**Pull request merged**